lib/ansible/modules/network/ios/ios_vlan.py

```diff
@@ -108,7 +108,6 @@
 
 
 def search_obj_in_list(vlan_id, lst):
-    obj = list()
```

Comment:
This change is not related to the PR purpose, so it's better to send it as a separate PR.

*__Pro tip:__ It's the best practice to avoid mixing unrelated changes into the same patch. Make atomic logical changes relating to the very thing you want to change, do other changes in separate PRs.*